LiteCore/Query/SQLiteQuery.cc

```diff
@@ -263,6 +263,15 @@ namespace litecore {
         {
             logInfo("Created on {Query#%u} with %llu rows (%zu bytes) in %.3fms",
                 query->objectRef(), rowCount, recording->data().size, elapsedTime*1000);
+
+            if (_iter[0u]) {
```

Comment:
Is this OK if there is nothing in the array?

Reply:
It's okay, just there is no columns i

Reply:
Oh, this _iter is for the columns. I see.